modules/os-datagrid-online-services-configuration/src/main/bash/profiles/caching-service.cli

```diff
@@ -18,6 +18,14 @@ end-if
 /subsystem=datagrid-infinispan/cache-container=clustered/distributed-cache=default:write-attribute(name=configuration, value=caching-service)
 /subsystem=datagrid-infinispan/cache-container=clustered/distributed-cache=memcachedCache:write-attribute(name=configuration, value=caching-service)
 
+# This is a temporary solution until we find out what the proper way is. Alternatively this could be implemented in
+# services.xml however we are not fully committed to OpenShift PING at this point.
```

Comment:
The auth code can also be moved to `services.xml`, but because of the delay getting things into JDG it's best if we keep with CLI for now and then update `services.xml` once things are closer to being finalized for initial release

Reply:
Good point, shall I add similar comment there?

Reply:
I say leave it as it is. It will be one of us updating the cli/templates in the not so distant future, so we will remember.

Reply:
Alrighty! Sounds good to me!